### DISCUSSION TITLE
mainnet sync err: miner id does not map to peer that sent message

### DISCUSSION THREAD

**Discussion by TonyS001** [filecoin-project/lotus#11099] [General Q&A]
### Checklist

- [X] This is **not** a security-related bug/issue. If it is, please follow please follow the [security policy](https://github.com/filecoin-project/lotus/security/policy).
- [X] I **have** searched on the [issue tracker](https://github.com/filecoin-project/lotus/issues) and the [lotus forum](https://github.com/filecoin-project/lotus/discussions), and there is no existing related issue or discussion.
- [X] I am running the [`Latest release`](https://github.com/filecoin-project/lotus/releases), the most recent RC(release canadiate) for the upcoming release or the dev branch(master), or have an issue updating to any of these.
- [X] I did not make any code changes to lotus.

### Lotus component

- [X] lotus daemon - chain sync
- [ ] lotus fvm/fevm - Lotus FVM and FEVM interactions
- [ ] lotus miner/worker - sealing
- [ ] lotus miner - proving(WindowPoSt/WinningPoSt)
- [ ] lotus JSON-RPC API
- [ ] lotus message management (mpool)
- [ ] Other

### Lotus Version

```text
1.23.2-dev+mainnet+git.517c0a53b
APIVersion:66816
```

### Describe the Bug

I start a mainnet node but can't sync. The log shows `miner id does not map to peer that sent message`.  
I attach the log file.
[lotus.log](https://github.com/filecoin-project/lotus/files/12111658/lotus.log)

### Logging Information

```text
`miner id does not map to peer that sent message`
```

**Comment by arajasek** [comment]
@TonyS001 Can you please share the results of `lotus sync status` and `lotus net peers` here?

**Comment by arajasek** [comment]
Moving this issue to Lotus Discussions for help and troubleshooting.

Closing Lotus issue ticket.

**Comment by jerry900314** [comment]
We downloaded the light version and encountered an issue: "Miner ID does not map to the peer that sent the message and cannot authenticate the message." {"err": "Miner ID does not map to the peer that sent the message", "errVerbose": "Miner ID does not map to the peer that sent the message."} : This issue should be resolved after syncing is completed?? Or any comment?
